gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/util/UnitTestConfig.java

```diff
@@ -68,11 +68,6 @@ public CustomObjectsApi customObjectsApi() {
         return new CustomObjectsApi(apiClient());
     }
 
-//    @Bean
-//    public AuthService authService() {
-//        return new AuthService();
```

Comment:
Is this temporary?

Reply:
No this is a permanent change. GaaS-rest no longer deals with authentication. All of the requests to rest are first filtered through an authentication sidecar. Essentially it sits in front of rest and intercepts all requests in order to authenticate. Therefore, if a request reaches rest it means that the user has already been authenticated. That's why in this PR different services/tests relating to authentication have been deleted from rest.